query-node/mappings/src/content/channel.ts

```diff
@@ -289,7 +181,7 @@ export async function content_ChannelOwnerRemarked(ctx: EventContext & StoreCont
   try {
     const decodedMessage = ChannelOwnerRemarked.decode(message.toU8a(true))
     const messageType = decodedMessage.channelOwnerRemarked
-    const contentActor = await convertContentActor(ctx.store, owner)
+    const contentActor = getcontentActor(channel.ownerMember, channel.ownerCuratorGroup)!
```

Comment:
No need for non-null assertation now. See the suggestion above this one.

```suggestion
    const contentActor = getContentActor(channel.ownerMember, channel.ownerCuratorGroup)
```